staging/src/k8s.io/pod-security-admission/metrics/metrics.go

```diff
@@ -16,9 +16,54 @@ limitations under the License.
 
 package metrics
 
+import (
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	admissionapi "k8s.io/pod-security-admission/admission/api"
+)
+
+const (
+	namespace = "pod_security"
+)
+
+var SecurityEvaluation = metrics.NewCounterVec(
+	&metrics.CounterOpts{
+		Namespace:      namespace,
+		Name:           "evaluations_total",
```

Comment:
Metrics name `pod_security_evaluations_total` is more concrete.

https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/2579-psp-replacement/README.md#monitoring